spec/teaspoon_env.rb

```diff
@@ -38,7 +38,7 @@
 
     # Specify a file matcher as a regular expression and all matching files will be loaded when the suite is run. These
     # files need to be within an asset path. You can add asset paths using the `config.asset_paths`.
-    suite.matcher = "{spec/javascripts,app/assets}/**/*_spec.{js,js.coffee,coffee}"
+    suite.matcher = "{spec/javascripts,app/assets}/*_spec.{js,js.coffee,coffee}"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.